packages/preview/secret/0.1.0/lib.typ

```diff
@@ -0,0 +1,9 @@
+A simple package to keep your secrets.
+
+#let redact(text, fill: black, height: 1em) = {
+  box(rect(fill: fill, height: height)[#hide(text)])
```

Comment:
This can simply be `box(fill: fill, height: height, hide(text))`.

Reply:
Good point, will update my code for it, thanks :)